sympy/geometry/polygon.py

```diff
@@ -403,7 +403,7 @@ def centroid(self):
             cy += v*(y1 + y2)
         return Point(simplify(A*cx), simplify(A*cy))
 
-    def second_moment_of_area(self,p=None):
+    def second_moment_of_area(self, p = None):
```

Comment:
What happens if a Point 3D is passed in? Any reason to worry about that?

Reply:
second moment and product moment of the area is 2D property of a figure. it is obvious p should be 2D although if p is like (x, y, z) then the function will use p as (x, y) only.

Reply:
@moorepants sir, will it create any issue?